src/main/java/com/example/animalshelter/model/AnimalHealthStatus.java

```diff
@@ -0,0 +1,11 @@
+package com.example.animalshelter.model;
+
+public enum AnimalHealthStatus {
+
+    UNKNOWN,
```

Comment:
Maybe a better option would be "not examined". Unknow says nothing.

Reply:
That's right, changed.